external/runtime/runtime.depproj

```diff
@@ -73,6 +75,43 @@
     <Exec Command="chmod +x $(TestHostRootPath)dotnet" Condition="'$(RunningOnUnix)' == 'true'"/>
   </Target>
 
+  <PropertyGroup>
+    <_CoreCLRPackageId>runtime.$(NugetRuntimeIdentifier).Microsoft.NETCore.Runtime.CoreCLR</_CoreCLRPackageId>
+    <_CoreCLRSymbolPackageUrl>https://dotnet.myget.org/F/dotnet-core/symbols/$(_CoreCLRPackageId)/$(CoreClrPackageVersion)</_CoreCLRSymbolPackageUrl>
+    <_CoreCLRSymbolPackageDestinationDirectory>$(IntermediateOutputPath)/symbolPackages/</_CoreCLRSymbolPackageDestinationDirectory>
+    <_CoreCLRSymbolPackageDestinationFile>$(_CoreCLRSymbolPackageDestinationDirectory)$(_CoreCLRPackageId).$(CoreClrPackageVersion).symbols.nupkg.zip</_CoreCLRSymbolPackageDestinationFile>
+    <_CoreCLRSymbolPackageUnzipFolder>$(_CoreCLRSymbolPackageDestinationDirectory)/$(_CoreCLRPackageId).$(CoreClrPackageVersion)</_CoreCLRSymbolPackageUnzipFolder>
+  </PropertyGroup>
+
+  <Target Name="DownloadCoreCLRSymbolPackage"
```

Comment:
We may end up needing a condition to disable this in some scenarios but for now I think we can always do it.